angular-workspace/projects/example-client-app/.eslintrc.js

```diff
@@ -1,43 +1,33 @@
 module.exports = {
     extends: '../../.eslintrc.js',
-    ignorePatterns: [
-        '!**/*'
-    ],
-    overrides: [
-        {
-            files: [
-                '*.ts'
+    ignorePatterns: ['**/environment*.ts'],
```

Comment:
can you do this to the specific environments folder? This match seems overly broad